clients/python/test/e2e/conftest.py

```diff
@@ -71,3 +72,28 @@ def sleeper(api_client: osparc.ApiClient) -> osparc.Solver:
         "simcore/services/comp/itis/sleeper", "2.0.2"
     )  # type: ignore
     return sleeper
+
+
+@pytest.fixture
+def sleeper_study_id(api_client: osparc.ApiClient) -> UUID:
+    """Simple sleeper study template which takes
+    as input a single file containing a single integer"""
+    _test_study_title = "sleeper_test_study"
+    study_api = osparc.StudiesApi(api_client=api_client)
+    for study in study_api.studies():
+        if study.title == _test_study_title:
+            return UUID(study.uid)
+    raise AssertionError(f"Could not find {_test_study_title}")
```

Comment:
check [`pytest.fail("Could not ...")`](https://docs.pytest.org/en/7.1.x/reference/reference.html?highlight=fail#pytest.fail)